cli/pom.xml

```diff
@@ -206,6 +206,11 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.json</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
What's this dep needed for? It looks like DMR is used to output JSON.

Reply:
This dependency is used on the test to parse the output of the DMR as a correctly formed JSON.

Reply:
Sorry I missed that part.